.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: etl2pcapng Build
+
+on:
+  push:
+    branches:
+    - master

```

Comment:
We should rename this to `main`.

Reply:
this == the primary branch, not just this string.

Reply:
Agreed. Do you remember the steps to do this?

Reply:
Merge this PR and then I can help with this.